dll/win32/rasdlg/lang/ro-RO.rc

```diff
@@ -1451,7 +1451,7 @@ BEGIN
     PUSHBUTTON "Reguli de apel", 1108, 193, 195, 65, 14
     CONTROL "", -1, "STATIC", SS_LEFTNOWORDWRAP | SS_SUNKEN | WS_GROUP, 10, 215, 248, 1
     DEFPUSHBUTTON "&Apelează", 1590, 8, 221, 60, 14, WS_GROUP
-    PUSHBUTTON "A&nulează", 1591, 71, 221, 60, 14
+    PUSHBUTTON "&Revocare", 1591, 71, 221, 60, 14
```

Comment:
Accelerator collision vs.
PUSHBUTTON "P&roprietăți", 1107, 134, 221, 60, 14
Check your Romanian Windows to find out which to change.